libzecale/circuits/null_hash_gadget.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-2020 Clearmatics Technologies Ltd
+//
+// SPDX-License-Identifier: LGPL-3.0+
+
+#ifndef __ZECALE_CIRCUIT_NULL_HASH_GADGET_HPP__
+#define __ZECALE_CIRCUIT_NULL_HASH_GADGET_HPP__
+
+namespace libzecale
+{
+
+/// A trivial hash gadget as a way to disable verification key hashing in
```

Comment:
`A trivial hash gadget as a way to disable verification key hashing` -> `A trivial hash gadget used for development purposes as a way to disable verification key hashing` ?

Reply:
Updated the phrasing of this comment.

Reply:
Thanks